api/tests/integration/environments/test_clone_environment.py

```diff
@@ -0,0 +1,214 @@
+import json
+
+from django.urls import reverse
+from rest_framework import status
+from tests.integration.helpers import get_env_feature_states_list_with_api
+
+from environments.permissions.models import UserEnvironmentPermission
+from features.models import (
+    Feature,
+    FeatureSegment,
+    FeatureState,
+    FeatureStateValue,
+)
+
+
+def test_clone_environment_returns_200(
```

Comment:
Do we need this test at all? We're already checking the response is 200 in the next test, right?